features/include/pcl/features/boundary.h

```diff
@@ -40,7 +40,6 @@
 
 #pragma once
 
-#include <pcl/features/eigen.h>
```

Comment:
Is it save to just remove this? The checks have passed, so there is nothing obvious missing, but that file contains `#include <Eigen/StdVector>`, which might not give errors when it is missing, but might still cause subtle problems? I think the safest way is to replace the includes you have removed (eigen.h / boost.h, more below)  with their respective contents (unless you are sure that a file does not need a specific include).

Reply:
I guess it should be verfied what headers are required and not just add all. 
If we just move all, we really haven't gained anything?

Reply:
This file (or the impl or the cpp) don't have any `vector<Eigen::X>` so I think it's safe to remove the include

Reply:
I believe their removal is safe. I checked the code and didn't find any usage for them.